### DISCUSSION TITLE
Rename file in channel form / rename file on upload

### DISCUSSION THREAD

**Discussion by zakirhpro** [ExpressionEngine/ExpressionEngine#3375] [Feature Requests]
I want to be able to change the filename and append their user ID to it but can't figure out how...the EE documentation doesn't provide any clues either.

Thanks

**Comment by intoeetive** [comment]
I feel like option to have the file automatically renamed on upload (based on pre-defined criterias or pattern) would make good feature request.

Converting to discussion
